app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1267,11 +1271,12 @@ private fun ThumbnailTile(
     openImageViewer: (url: String) -> Unit,
 ) {
     postView.post.url?.also { url ->
+        val postType = getPostType(url)
 
         val postLinkPicMod = Modifier
             .size(POST_LINK_PIC_SIZE)
             .clickable {
-                if (isImage(url)) {
+                if (postType != PostType.Link) {
```

Comment:
I prefer `postType == PostType.Image`

Reply:
Not the way I was imagining it, I was going to add a separate videoViewer. If imageViewer would support videos I would have rather that it be named mediaViewer. Or something generic.

Reply:
I was also imagining a separate video viewer. This bit of code doesn't really matter tbh until we integrate the image viewer, but I can change it to ==.

Reply:
I was fine with the hard coding tbh. But since we already have this implemented with the theme might as well keep it. @twizmwazin wanna be the tie braker?